.gitignore

```diff
@@ -1,57 +1,101 @@
-# Package Files #
-*.jar
-*.war
-*.ear
+#
```

Comment:
I don't think we need to have (c) statement in `.gitignore`

Reply:
Yep, I'll remove this one from here and I created the issue https://github.com/SpineEventEngine/config/issues/163 to refine where we'd like to have (c) and where not.